src/StanfordBehat/DrupalExtension/Context/SwsMinkContext.php

```diff
@@ -162,7 +179,7 @@ protected function waitForSelector(callable $lambda, $selector, $attempts = 30)
       sleep(1);
     }
 
-    throw new ElementNotVisible('The element is not available ' . $last_message);
+    throw new ElementNotVisible("The element is$not available $last_message");
```

Comment:
```suggestion
    throw new ElementNotVisible("The element is $not available $last_message");
```

Reply:
having the space produces double spaces sometimes if `$not = ''`

Reply:
not really a big deal though. :man_shrugging:

Reply:
Ah right. Ah well. Whatevs.